packages/react-components/src/types/OnRender.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  IPersonaStyleProps,
+  IPersonaStyles,
+  IStyleFunctionOrObject,
+  PersonaPresence,
+  PersonaSize
+} from '@fluentui/react';
+import { PlaceholderProps } from '../components/VideoTile';
+
+/**
+ * A custom rendered callback that allows users to customize the rendering of a Persona Component.
+ * @param {string} userId - The Azure Communication Services ID of the user whose Persona is being rendered.
+ * @param {object} options - Default Persona options such as `size`.
+ */
+export type OnRenderAvatarType = (
+  userId?: string,
+  options?: {
+    /** Only show Coin and Initials  */
+    hidePersonaDetails?: boolean;
+    /** Text color of initials inside the coin  */
+    initialsTextColor?: string;
+    /** User status  */
+    presence?: PersonaPresence;
+    /** Preset Persona Size number  */
+    size?: PersonaSize;
+    /** Persona coin size in pixels  */
+    coinSize?: number;
+    /** Display name to be used in Persona  */
+    text?: string;
+    styles?: IStyleFunctionOrObject<IPersonaStyleProps, IPersonaStyles>;
+  },
+  /**
```

Comment:
Ah, options is directly passable!

Going in the opposite direction than my earlier comment: Perhaps we should simply extract the options type to a named type and use that instead of `PlaceholderProps` below?

Reply:
Ah yes, that was the idea originally. I think I forgot to do it

Reply:
Oh right. The signature of options and placeholder props are different. They can't be directly passed